packages-next/fields/src/types/image/tests/test-fixtures.ts

```diff
@@ -0,0 +1,271 @@
+import path from 'path';
+import fs from 'fs-extra';
+// @ts-ignore
```

Comment:
Why is this ts-ignored?

Reply:
Otherwise we end up with:

```
Module '"graphql-upload"' has no exported member 'Upload'.
```

The code works, so I'm not keen on tracking down why typescript is complaining here, but if you know a better solution I'm all ears 👍